app_datetime/models.py

```diff
@@ -0,0 +1,7 @@
+from django.db import models
+from django.utils.translation import gettext as _
+
+
+class DateTime(models.Model):
```

Comment:
`DataTime` is generally a field type. Maybe use `DateTimeRecord`

Reply:
Done